roles/tuned/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 tuned_etc_directory: '/etc/tuned'
 tuned_templates_source: '../templates'
+openshift_fs_inotify_max_user_watches: 65536
```

Comment:
@sjenning  65536 here is different from 
fs.inotify.max_user_watches = 524288 in comment
https://bugzilla.redhat.com/show_bug.cgi?id=1559061#c3
I am confused.
Is this intended? 
Thanks.

Reply:
I just put 65536 as a default value as in my understanding 65536 is the default value of  fs.inotify_max_user_watches. If you want to change default value and have valid reasons, I am fine with it.

Reply:
got it. Thanks for explanation.